src/contractTest/java/uk/gov/hmcts/reform/orgrolemapping/OrgRoleMappingConsumerTestForCRD.java

```diff
@@ -0,0 +1,293 @@
+                        .post(mockServer.getUrl() + CRD_GET_ROLE_ASSIGNMENT_URL)
+                        .then()
+                        .log().all().extract().asString();
+
+        JSONObject jsonResponse = new JSONObject(actualResponseBody);
+        JSONArray crdResponse = (JSONArray)jsonResponse.get("");
+        JSONObject firstRecord = (JSONObject)crdResponse.get(0);
+        JSONArray roleObject = (JSONArray)firstRecord.get("role");
+        JSONObject roleRecord = (JSONObject)roleObject.get(0);
+        assertThat(roleRecord.get("role"), equalTo("senior-tribunal-caseworker"));
+
+        JSONObject secondRecord = (JSONObject)crdResponse.get(1);
+        JSONArray roleObject2 = (JSONArray)secondRecord.get("work_area");
+        JSONObject roleRecord2 = (JSONObject)roleObject2.get(0);
+        assertThat(roleRecord2.get("service_code"), equalTo("BFA1"));
+    }
+
+    private DslPart createCRDResponse() {
+        return newJsonBody(o -> o
+                .minArrayLike("", 1, 1,
+                    crdResponse -> crdResponse
+                    .stringType("id", "91e07fe0-9575-472b-bd1f-33be2944c1f4")
+                    .stringValue("idamRoles", null)
+                    .stringValue("first_name", "testFirstname")
+                    .stringValue("last_name", "TestSurname")
+                    .stringValue("email_id", "sam.test@gmail.com")
+                    .stringValue("regionId", "1")
+                    .stringValue("region", "National")
+                    .array("base_location", (bl) -> {
+                        bl.object((bo) -> {
+                            bo
+                                .stringType("location_id", "2191654")
+                                .stringType("location", "Aberdeen Tribunal Hearing Centre")
+                                .booleanValue("is_primary", true);
+                        });
+                    })
+                    .stringValue("user_type_id", "1")
+                    .stringValue("user_type", "HMCTS")
+                    .array("role", (r) -> {
+                        r.object((ro) -> {
+                            ro
+                                .stringType("role_id", "1")
+                                .stringType("role", "senior-tribunal-caseworker")
+                                .booleanValue("is_primary", true);
+                        });
+                    })
+                    .array("work_area", (wa) -> {
+                        wa.object((wo) -> {
+                            wo
+                                .stringType("service_code", "BFA1")
+                                .stringType("area_of_work", "1");
+                        });
+                    })
+                    .booleanValue("delete_flag", false)
+                )
+        ).build();
+    }
+
+    private DslPart createCRDMultipleUsersResponse() {
+        return newJsonBody(o -> o
+                .minArrayLike("", 2, 2,
```

Comment:
Provide the name of array here instead of "". As, it will easy to understand and iterate.

Reply:
At the moment, CRD response doesn't have any name for this. Will update once we got the name from CRD team.